src/test/java/io/jenkins/plugins/httpclient/RobustHTTPClientTest.java

```diff
@@ -24,19 +24,48 @@
 
 package io.jenkins.plugins.httpclient;
 
-import static org.junit.Assert.*;
+import static org.hamcrest.MatcherAssert.*;
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.assertThrows;
 
 import java.net.URL;
+import org.junit.Assume;
 import org.junit.Test;
 
 public class RobustHTTPClientTest {
-
     @Test
     public void sanitize() throws Exception {
-        assertEquals("http://x.com/some/path", RobustHTTPClient.sanitize(new URL("http://x.com/some/path")));
-        assertEquals(
-                "https://x.com/some/path?…", RobustHTTPClient.sanitize(new URL("https://x.com/some/path?auth=s3cr3t")));
-        assertEquals(
-                "https://…@x.com/otherpath", RobustHTTPClient.sanitize(new URL("https://user:s3cr3t@x.com/otherpath")));
+        assertThat(
+                RobustHTTPClient.sanitize(new URL("http://example.com/some/long/path")),
+                is("http://example.com/some/long/path"));
```

Comment:
> Also revises the existing test to use example.com instead of x.com since there is now a site with the x.com domain name.

this is at odds here, if the rationale here was that `x.com` now exists so use something that does not exist, then we can not use `example.com` as that exists (and is reserved).   in other words you should use a different reserved domain (or TLD) that reserved but should not exist.?

e.g. `foo.invalid`

Reply:
The switch from x.com to example.com had multiple motivations.  I didn't mention all the motivations in the commit message.  I should have done that.

It was intended to intentionally switch to a domain name that is [reserved for documentation purposes](https://en.wikipedia.org/wiki/Example.com).  The test is documenting that the sanitization behaves as expected.

It was intended to make the URL string longer so that all three of the assertions the tests would format the same way.

It was intended to stop using a domain that now exists and is a redirect to twitter.com.

I'm fine with switching to another domain if that makes the test clearer.